zathura/commands.c

```diff
@@ -410,8 +411,10 @@ bool cmd_search(girara_session_t* session, const char* input, girara_argument_t*
   /* set search direction */
   zathura->global.search_direction = argument->n;
 
-  unsigned int number_of_pages     = zathura_document_get_number_of_pages(zathura->document);
-  unsigned int current_page_number = zathura_document_get_current_page_number(zathura->document);
+  unsigned int number_of_pages          = zathura_document_get_number_of_pages(zathura->document);
+  unsigned int current_page_number      = zathura_document_get_current_page_number(zathura->document);
+  zathura->global.total_search_results  = 0;
+  zathura->global.current_search_result = 0;
```

Comment:
Add an empty line in between and avoid the reformatting of the two lines above.

Reply:
Please skip the reformatting of lines 413 and 414.

Reply:
@sebastinas Done